docs/content/providers/kubernetes-ingress.md

```diff
@@ -64,23 +64,36 @@ as is a common pattern in the kubernetes ecosystem.
 When using a single instance of Traefik with LetsEncrypt, no issues should be encountered,
 however this could be a single point of failure.
 Unfortunately, it is not possible to run multiple instances of Traefik 2.0 with LetsEncrypt enabled,
-because there is no way to ensure that the correct instance of Traefik will receive the challenge request, and subsequent responses.
+because there is no way to ensure that the correct instance of Traefik receives the challenge request, and subsequent responses.
 Previous versions of Traefik used a [KV store](https://doc.traefik.io/traefik/v1.7/configuration/acme/#storage) to attempt to achieve this,
-but due to sub-optimal performance was dropped as a feature in 2.0.
+but due to sub-optimal performance that feature was dropped in 2.0.
 
-If you require LetsEncrypt with HA in a kubernetes environment,
+If you need LetsEncrypt with HA in a kubernetes environment,
 we recommend using [Traefik Enterprise](https://traefik.io/traefik-enterprise/) where distributed LetsEncrypt is a supported feature.
 
-If you are wanting to continue to run Traefik Community Edition,
+If you want to keep using Traefik Community Edition,
```

Comment:
```suggestion
If you want to keep using Traefik Proxy,
```
I'm not sure this gets the point across the same way that "community edition" does, but this is the branding we're using now.